spec/unit/build.spec.js

```diff
@@ -44,11 +44,17 @@ function createConfigParserMock(winVersion, phoneVersion) {
         return {
             getPreference: function(prefName) {
                 switch (prefName) {
-                    case 'windows-target-version':
+                    case 'WindowsTargetVersion':
                         return winVersion;
-                    case 'windows-phone-target-version':
```

Comment:
`windows-phone-target-version` should continue to work. See `config.getWindowsPhoneTargetVersion` implementation
```
 getWindowsPhoneTargetVersion: function() {
...
+        var explicitPreference = this.getPreference('WindowsPhoneTargetVersion') || this.getPreference('windows-phone-target-version');
```